features/post.feature

```diff
@@ -473,4 +474,23 @@ Feature: Manage WordPress posts
     Then STDOUT should contain:
       """
       2005-01-24 09:52:00
-      """
\ No newline at end of file
+      """
+
+  @require-sqlite
+  Scenario: Publishing a post and setting a date succeeds if the edit_date flag is passed.
+    Given a WP install
+
+    When I run `wp post create --post_title='test' --porcelain`
+    Then save STDOUT as {POST_ID}
+
+    When I run `wp post update {POST_ID} --post_date='2005-01-24T09:52:00.000Z' --post_status='publish' --edit_date=1`
+    Then STDOUT should contain:
+      """
+      Success:
+      """
+
+    When I run `wp post get {POST_ID} --field=post_date`
+    Then STDOUT should contain:
+      """
+      2005-01-24T09:52:00.000Z
```

Comment:
@swissspidy Seems to be a bug (in the SQLite plugin?) that the date is returned in a different format than with MySQL. What's the reasoning behind adding a separate test here, instead of fixing the behavior?

Reply:
Yes it seems to be a bug, but in the SQLite plugin, so it should be fixed there. The thinking was that this test documents the existing behavior in the meantime, until that bug is fixed.

Reply:
Bug filed here: https://github.com/WordPress/sqlite-database-integration/issues/52

Reply:
We can also just remove this extra test for now, and simply skip the original one for SQLite until this bug is resolved. Don't have a strong opinion.

Reply:
@schlessera What's your preference here?

Reply:
Lets keep the separate branch for now to unblock this but add a comment pointing to the bug report in the plugin.

Reply:
Done!